contracts/InterestRate/StableRate.sol

```diff
@@ -0,0 +1,240 @@
+        uint256 util = utilizationRate(cash, totalBorrows, reserves);
+        uint256 loanRate = stableLoanRate(stableBorrows, totalBorrows);
+        uint256 excessLoanRate = calculateLoanRateDiff(loanRate);
+
+        if (util <= kink) {
+            return
+                ((util * multiplierPerBlock) / BASE) +
+                baseRatePerBlockForStable +
+                ((util * multiplierPerBlockForStable) / BASE) +
+                ((stableRatePremium * excessLoanRate) / BASE);
+        } else {
+            uint256 excessUtil = util - kink;
+            return
+                ((kink * multiplierPerBlock) / BASE) +
+                baseRatePerBlockForStable +
+                ((kink * multiplierPerBlockForStable) / BASE) +
+                ((excessUtil * jumpMultiplierPerBlockForStable) / BASE) +
+                ((stableRatePremium * excessLoanRate) / BASE);
+        }
+    }
+
+    /**
+     * @notice Internal function to update the parameters of the interest rate model
+     * @param _baseRatePerBlockForStable The approximate target base APR, as a mantissa (scaled by BASE)
+     * @param _multiplierPerBlock The rate the interest rate increases per utilization (scaled by BASE)
+     * @param _multiplierPerBlockForStable The rate of increase in interest rate with respect to utilization (scaled by BASE)
+     * @param _jumpMultiplierPerBlockForStable The multiplierPerBlock after hitting a specified utilization point
+     * @param _kink The utilization point where the jump multiplier is applied
+     * @param _stableRatePremium The multiplierPerBlock after hitting a specified utilization point
+     * @param _optimalStableLoanRate Optimal stable loan rate percentage.
+     */
+    function updateStableRateModelInternal(
+        uint256 _baseRatePerBlockForStable,
+        uint256 _multiplierPerBlock,
+        uint256 _multiplierPerBlockForStable,
+        uint256 _jumpMultiplierPerBlockForStable,
+        uint256 _kink,
+        uint256 _stableRatePremium,
+        uint256 _optimalStableLoanRate
+    ) internal {
+        baseRatePerBlockForStable = _baseRatePerBlockForStable / blocksPerYear;
+        multiplierPerBlock = (_multiplierPerBlock * BASE) / (blocksPerYear * _kink);
+        multiplierPerBlockForStable = (_multiplierPerBlockForStable * BASE) / (blocksPerYear * _kink);
+        jumpMultiplierPerBlockForStable = _jumpMultiplierPerBlockForStable / blocksPerYear;
+        kink = _kink;
+        stableRatePremium = _stableRatePremium;
+        optimalStableLoanRate = _optimalStableLoanRate;
+
+        emit NewStableInterestParams(
+            baseRatePerBlockForStable,
+            multiplierPerBlock,
+            multiplierPerBlockForStable,
+            jumpMultiplierPerBlockForStable,
+            kink,
+            stableRatePremium,
+            optimalStableLoanRate
+        );
+    }
+
+    /**
+     * @notice Calculates the stable loan rate of the market: `stableBorrows / totalBorrows`
```

Comment:
Wrong comment

Reply:
Fixed. Also changes the method name to get the better understanding. Can verify the changes at [commit](https://github.com/VenusProtocol/isolated-pools/pull/132/commits/19cb54d75f30e364c0bb78f599cdae3da32a3838).